tools/scripts/stm32.mk

```diff
@@ -133,9 +141,9 @@ SIZE=arm-none-eabi-size
 
 .PHONY: $(BINARY).openocd
 $(BINARY).openocd:
-	@echo source [find interface/stlink-dap.cfg] > $(BINARY).openocd
```

Comment:
let's not change anything in this .openocd file for now

you're adding vscode support, so no need to touch the `make run` and `make debug` stuff

in any case, this change would make `make run` not work with nucleo boards anymore

Reply:
Sorry for the confusion. I should have squashed the commits to one. This part is outdated. I will squash and resubmit.